src/archanaNJun20/BookMeetingRoom.java

```diff
@@ -0,0 +1,112 @@
+package archanaNJun20;
+
+public class BookMeetingRoom {
+	String meetingRoom;
+	double hours=0.0;
+	boolean nalandaFlag=false;
+	boolean takshashilaFlag=false;
+	boolean agraFortFlag=false;
+	boolean pratapGadhFlag=false;
+
+	void bookMeetingRoom(){
+		if(nalandaFlag==false){
+			meetingRoom="Nalanda";
+			hours=1;
+			displayInfo();
+			nalandaFlag=true;
+		}
+		else{
+			System.out.println("-->The meeting room  is already booked.");
+		}
+	}
+	void bookMeetingRoom(String meetingRoom){
+		if(!(meetingRoom.equals("Nalanda"))){
+			if(meetingRoom.equals("Takshashila")&&takshashilaFlag==false){
+				this.meetingRoom="Takshashila";
+				hours=1;
+				displayInfo();
+				takshashilaFlag=true;
+			}
+			else if(meetingRoom.equals("AgraFort") &&agraFortFlag==false){
+				this.meetingRoom="AgraFort";
+				hours=1;
+				displayInfo();
+				agraFortFlag=true;
+			}
+			else if(meetingRoom.equals("PratapGadh")&&pratapGadhFlag==false){
+				this.meetingRoom="PratapGadh";
+				hours=1;
+				displayInfo();
+				pratapGadhFlag=true;
+			}
+			else{
+				System.out.println(meetingRoom +"-->The meeting room is already booked.");
+			}
+		}
+		else{
+			System.out.println(meetingRoom +" is Invalid meeting Room ");
```

Comment:
This statement will show "Nalanda is invalid meeting room" if user passes "Nalanda" as a value.
for Invalid meeting room you will have to add one more if condition to check all the valid meeting room names

Reply:
The code is updated as per your suggestion.

Reply:
it will still show "Nalanda is invalid meeting room"
you can write like this:

if(meetingRoom.equals("Nalanda"){
SOP("Nalanda is already booked"); Or you can call BookMeeetingRoom() method from here
}
else if(meetingRoom.equals("Takshashila") || meetingRoom.equals("AgraFort") || meetingRoom.equals("PratapGadh")){

}
else{
SOP("Invalid Room");
}

Also update the Output screenshot